ml-agents/mlagents/trainers/components/reward_signals/entropy/signal.py

```diff
@@ -0,0 +1,22 @@
+from mlagents.trainers.components.reward_signals import RewardSignal
+from mlagents.trainers.policy import Policy
+
+
+class EntropySignal(RewardSignal):
```

Comment:
`EntropyRewardSignal`?

Reply:
Am going to remove the Entropy reward signal as it is handled already within the PPO (as entropy regularization). It also doesn't generalize well to SAC.